Core/tooling/build-final.js

```diff
@@ -21,43 +15,13 @@ const fixNodeImports = (src) => src
   .replace(/require\(['"]minimalistic-assert['"]\)/g, 'dereq_minimalistic_assert')
   .replace(/require\(['"]inherits['"]\)/g, 'dereq_inherits')
   .replace(/require\(['"]asn1\.js['"]\)/g, 'dereq_asn1');
-const nodeDepsSrc = fixNodeImports(fs.readFileSync(path.nodeDepsBundle).toString())
-const nodeDevDepsSrc = fixNodeImports(fs.readFileSync(path.nodeDevDepsBundle).toString())
-const nodeEntrypointSrc = fs.readFileSync(path.nodeEntrypointBundle).toString().replace("'[BUILD_REPLACEABLE_VERSION]'", 'APP_VERSION');
```

Comment:
The issue is likely somewhere in this file. To give you a clearer view, there were three build types:

```ts
  finalDev: 'build/final/flowcrypt-android-dev.js',
  finalIos: 'build/final/flowcrypt-ios-prod.js',
  finalAndroid: 'build/final/flowcrypt-android-prod.js',
```

This was the purpose of each one of them:

## `finalIos`

This is run in iOS through JavaScriptCore - which is a "bare" js engine in the sense that it doesn't have a lot of the stuff that we are used to in NodeJS or browser js runtimes. No `setTimeout`, no `crypto.getRandomBytes` and loads of other things. So we have to polyfill a lot of this one way or another, and bundle everything into one javascript file, including all dependencies.

## `finalAndroid`

This was run on Android NodeJS, but it's also non-standard. There was no node_modules to use on the Android phone, so we also have to bundle everything. But because it's NodeJS, there in fact are many more utilities already present in the runtime, so we in fact bundle fewer things. But on the other hand, we had to include a http server.

## `finalDev`

This was run on local machine for tests. The runtime is NodeJS, but this time we can use dependencies from node_modules where available. Included a http server.

-----------------------------------------

Now after this PR, I expect we will have the following:

`finalIos` - unchanged

`finalDev` - for NodeJS running locally, taking advantage of node_modules where possible as before, but no http server

-----------------

Therefore, you should not expect to be able to run tests on the `bare` (meaning ios) version. You'll want to run it on the dev version, but a modified one - without the http server, but with unchanged handling of dependencies, so that you don't have to go down this rabbit hole that I already went down years ago to get this to work.

I suppose this line will be needed still:

```ts
 const nodeDevDepsSrc = fixNodeImports(fs.readFileSync(path.nodeDevDepsBundle).toString())
```